sourcegraph/sourcegraph
# nit fix: description of access token ins sourcegraph VS Code Extensions in VS Code settings

**TL;DR;**

* Fixed Description of `sourcegraph.accessToken` in VS Code Settings File.
* Updated the docs about instructions to get the access token.

---

## Description

This PR aims to fix the following URL in VS Code Extension Settings:

![image](https://user-images.githubusercontent.com/55068936/185790404-2122cbc9-60f3-4fa2-afe7-8b64b47a942f.png)

& also, apart from ` ${SOURCEGRAPH_URL}` further URL is incorrect now. `https://sourcegraph.com/users/settings/tokens` redirects to 404: 

![image](https://user-images.githubusercontent.com/55068936/185790482-c69dbae4-425b-4544-b9c4-091406121c65.png)

Correct URL is: `https://sourcegraph.com/users/<sourcegraph-username>/settings/tokens`

## Test plan

- Change current directory to `client/vscode`
- Install Dependencies
- Build & Generate a `.vsix` file
- Install the file and see the difference
